test/e2e/alertmanager_test.go

```diff
@@ -482,8 +482,8 @@ func TestAlertmanagerDataReplication(t *testing.T) {
 	}
 }
 
-// The Alertmanager API should be protected by the OAuth proxy.
-func TestAlertmanagerOAuthProxy(t *testing.T) {
+// The Alertmanager API should be protected by authentication/authorization.
+func TestAlertmanagerAPI(t *testing.T) {
```

Comment:
I don't see how the test is checking the auth

Reply:
(not related to the PR)

Reply:
cc @simonpasquier if you missed this (I forgot o hold)
I could take care of it if you want.